src/Utilities/sets.jl

```diff
@@ -1,14 +1,32 @@
-const DimensionUpdatableSets = Union{MOI.Reals,
-                                     MOI.Zeros,
-                                     MOI.Nonnegatives,
-                                     MOI.Nonpositives}
 """
-    updatedimension(s::AbstractVectorSet, newdim)
+    supports_dimension_update(S::Type{<:MOI.AbstractVectorSet})
```

Comment:
Since `supports_dimension_update` and `update_dimension` are referenced in the definition of `delete`, they should be in MOI instead of MOIU. These functions are a part of the interface for defining a set.